app/src/main/java/com/yeonae/chamelezone/view/login/ConfirmEmailFragment.kt

```diff
@@ -37,9 +37,9 @@ class ConfirmEmailFragment : Fragment() {
 
     companion object {
         private const val EMAIL = "email"
-        fun newInstance(email: String) = ConfirmEmailFragment().apply {
+        fun newInstance(email: MutableList<String>) = ConfirmEmailFragment().apply {
             arguments = Bundle().apply {
-                putString(EMAIL, email)
+                putString(EMAIL, email.toString())
```

Comment:
```suggestion
                putStringArray(EMAIL, emails)
```
list를 string으로 변환해서 그걸 또 잘라서 list로 사용하는 상황이
부적절한 것 같아요.

Reply:
수정했습니당